react-app/src/hooks/page-hooks/useProductsAndServices.js

```diff
@@ -12,6 +12,7 @@ export default function useProductsAndServices() {
   const [categories, setCategories] = React.useState([]);
   const [taxOptions, setTaxOptions] = React.useState([]);
   const [salesAccounts, setSalesAccounts] = React.useState([]);
+  const [currencies, setCurrencies] = React.useState([]);
```

Comment:
consider having a dedicated component for fetching redudant stuff like this
```suggestion
import React from "react";
import axios from "axios";

export function useCurrencies() {
  const [currencies, setCurrencies] = React.useState([]);
  const [loading, setLoading] = React.useState(true);
  const [error, setError] = React.useState(null);

  React.useEffect(() => {
    axios
      .get("/accounting/currency")
      .then((res) => {
        setCurrencies(res.data);
        setLoading(false);
      })
      .catch((err) => {
        setError(err);
        setLoading(false);
      });
  }, []);

  return { currencies, loading, error };
}
```